ddp-workspace/projects/ddp-dsm-ui/src/app/abstraction-field/abstraction-field.component.ts

```diff
@@ -308,7 +309,7 @@ export class AbstractionFieldComponent implements OnInit {
           },
           error: err => {
             if (err._body === Auth.AUTHENTICATION_ERROR) {
-              this.router.navigate([ Statics.HOME_URL ]);
```

Comment:
Was this code assuming (rightly or wrongly?) that `HOME_URL` would handle logout properly?

Reply:
Exactly.